src/securityinsight/azext_securityinsight/azext_metadata.json

```
@@ -0,0 +1,4 @@
+{
+    "azext.isExperimental": true,
+    "azext.minCliCoreVersion": "2.37.0"
```

Please waiting for the release of "2.38.0"
```suggestion
    "azext.minCliCoreVersion": "2.38.0"
```

---

good catch👍